control-plane/agents/src/bin/core/main.rs

```diff
@@ -88,13 +88,42 @@ pub(crate) struct CliArgs {
     /// If `None` do not limit the number of rebuilds.
     #[clap(long)]
     max_rebuilds: Option<NumRebuilds>,
+
+    #[clap(flatten)]
+    thin_args: ThinArgs,
 }
 impl CliArgs {
     fn args() -> Self {
         CliArgs::parse()
     }
 }
 
+/// Cluster wide thin provisioning parameters.
+#[derive(Debug, clap::Parser, Clone)]
+pub(crate) struct ThinArgs {
+    /// The allowed pool commitment limit when dealing with thin provisioned volumes.
+    /// Example: If the commitment is 250 and the pool is 10GiB we can overcommit the pool
+    /// up to 25GiB (create 2 10GiB and 1 5GiB volume) but no further.
+    #[clap(long, env = "POOL_COMMITMENT_%", value_parser = value_parse_percent, default_value = "250%")]
+    pool_commitment: u64,
+    /// When creating pool replicas for a new volume, each replica pool must have at least this
```

Comment:
`When creating pool replicas for a new volume` -> `When creating replicas for a new volume` ?